src/main/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/entity/Match.java

```diff
@@ -0,0 +1,55 @@
+package ch.uzh.ifi.hase.soprafs22.screwyourneighborserver.entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import java.util.ArrayList;
+import java.util.Collection;
+import javax.persistence.*;
+
+@Entity
+public class Match {
+  @Id @GeneratedValue private Long id;
+
+  private int match_number;
```

Comment:
its java, not python: matchNumber.
Dass hier google java format nicht reklamiert...